7-layouts/styles.css

```
@@ -0,0 +1,145 @@
+/*Mobile design*/
+.articles {
+    display: flex;
+    flex-flow: column wrap;
+    margin-top: 40px;
+}
+
+.articles .animal {
+    display: flex;
+    flex-direction: column-reverse;
+    height: auto;
+    margin-bottom: 35px;
+}
+
+.articles .animal h3 {
+    background-color: rgba(88, 88, 88, 0.281);
+    font-size: 25px;
+    font-weight: 10;
+    position: absolute;
+    margin: 0;
+    width: 100%;
+}
+
+.articles .animal .animal-img {
+    display: block;
+    height: auto;
+    width: 100%;
+}
+
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.header section:first-child {
+    align-items: center;
+    background-color: #927811;
+    display: flex;
+    height: 40px;
+    padding-left: 10px;
+}
+
+.header section:first-child h1 {
+    color: white;
+    margin-left: 20px;
+}
+
+.header section:first-child img {
+    height: 30px;
+    width: 30px;
+}
+
+.header section:nth-child(2) {
+    background: url("assets/main-picture.jpg");
+    background-position: center center;
+    background-size: cover;
+    color: white;
+    font-family: Calibri;
+    height: 300px;
+    text-align: justify;
+}
+
+.header section:nth-child(2) h2 {
+    font-size: 50px;
+    margin: 0;
+}   
+
+.header section:nth-child(2) h2, .header section:nth-child(2) p {
+    padding-left: 30px;
+    padding-right: 40px;
+}
+
+.header section:nth-child(2) p {
+    font-size: 20px;
+}
+
+/*Tablet Design*/
+@media screen and (min-width: 600px) {
+    .articles {
+        flex-flow: row wrap;
+        justify-content: space-around;
+    }
+
+    .articles .animal {
+        align-self: center;
+        margin-bottom: 40px;
+        width: 400px;
+    }
+
+    .articles .animal h3 {
+        font-size: 35px;
+        position: relative;
+    }
+
+    .header section:first-child {
+        height: 80px;
+    }
+
+    .header section:first-child h1 {
+        font-size: 40px;
+    }
+
+    .header section:first-child img {
+        height: 40px;
+        width: 40px;
+    }
+
+    .header section:nth-child(2) {
+        height: 550px;
+    }
+
+    .header section:nth-child(2) h2 {
+        font-size: 100px;
+        padding-left: 50px;
+        padding-top: 5px;
+    }
+    
+    .header section:nth-child(2) p {
+        font-size: 35px;
+        padding-left: 50px;
+        padding-right: 60px;
+    }
+}
+
+/*Desktop Design*/
+@media screen and (min-width: 1024px) {
+    .articles .animal {
+        width: 427px;
+    }
+
+    .header section:nth-child(2) {
+        align-items: center;
+        display: flex;
+        justify-content: center;
+    }
+
+    .header section:nth-child(2) h2 {
+        padding: 0 0 0 50px;
+    }
+    
+    .header section:nth-child(2) p {
+        font-size: 45px;
+        padding: 0px 40px 0px 50px;
```

```suggestion
        padding: 0 40px 0 50px;
```
0px is equals to 0, no matter the unit.